src/SourceBuild/content/Directory.Build.targets

```diff
@@ -2,14 +2,18 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(SkipArcadeSdkImport)' != 'true'" />
 
-  <Target Name="DetermineSourceBuiltSdkVersion">
+  <Target Name="GetDotNetSdkArchive">
     <PropertyGroup>
-      <SdkFilenamePrefix>dotnet-sdk-</SdkFilenamePrefix>
+      <DotNetSdkFilenamePrefix>dotnet-sdk-</DotNetSdkFilenamePrefix>
     </PropertyGroup>
+
     <ItemGroup>
-      <SdkTarballItem Include="$(ArtifactsAssetsDir)Sdk/**/$(SdkFilenamePrefix)*$(ArchiveExtension)" />
+      <SdkArchiveItem Include="$(ArtifactsAssetsDir)Sdk/**/$(DotNetSdkFilenamePrefix)*$(ArchiveExtension)" />
     </ItemGroup>
+  </Target>
 
+  <Target Name="DetermineSourceBuiltSdkVersion"
+          DependsOnTargets="GetDotNetSdkArchive">
```

Comment:
Why do we need a separate target for this?

Reply:
While I think using the archive file name is fine, we could consider another method for extracting versions if we find we keep doing this kind of thing. We could take the vertical manifest and find a package with a matching version. This is how Maestro dependency flow typically works for assets. If installer wants to download aspnetcore's runtime archive, it will find an aspnet runtime package that will have a matching version, use that in its Version.Details.xml, then construct the file name off the version property.

The SDK doesn't produce a "sentinel" package today in the VMR. The regular official build does (`Microsoft.Dotnet.Sdk.Internal`).